modules/ocf_irc/manifests/services.pp

```diff
@@ -0,0 +1,37 @@
+class ocf_irc::services {
+  package { 'anope':; }
+
+  service { 'anope':
+    require => [Package['anope'], Service['inspircd']],
+  }
+
+  $passwords = parsejson(file("/opt/puppet/shares/private/${::hostname}/services-passwords"))
+
+  $root_nicks = ['waf', 'nattofriends', 'ckuehl', 'jvperrin', 'mattmcal']
```

Comment:
Alright, I just used `waf` because that was the config from before (current anope server), but adding more is fine.